### PR TITLE
Feat/install docs

### PR DESCRIPTION
**Pull request by andricDu**

### CODE REVIEW

**Comment by andricDu** [comment]
https://github.com/overture-stack/ego/issues/556

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+total 12
+drwxr-xr-x  3 root   root   4096 Mar 11 18:56 .
+drwxr-xr-x 19 root   root   4096 Mar 11 18:14 ..
+drwxrwxr-x  8 ubuntu ubuntu 4096 Mar 11 18:51 ego-4.1.0
+lrwxrwxrwx  1 root   root      9 Mar 11 18:56 ego-current -> ego-4.1.0
+```
+
+### Database Configuration
+
+The directory structure inside of the Ego directory is self explainatory:
```
Comment (b-f-chan) [inline INSTALL.md:52]: Spelling

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+INFO   | jvm 1    | 2021/03/11 19:35:02 | 2021-03-11 19:35:02,492 [WrapperSimpleAppMain] INFO  o.s.b.w.e.t.TomcatWebServer - Tomcat started on port(s): 8081 (http) with context path ''
+INFO   | jvm 1    | 2021/03/11 19:35:02 | 2021-03-11 19:35:02,497 [WrapperSimpleAppMain] INFO  b.o.e.AuthorizationServiceMain - Started AuthorizationServiceMain in 22.659 seconds (JVM running for 24.385)
+```
+
+## Auth Setup
+Now that Ego is up and running we want to configure the first user and application that can use Ego for Authorization.
+
+### Identity Provider
+
+For the identity provider of your choosing, find the relevant section in the `application.yml` configuration file and the client id and secret you have configured with that IdP. For example is Google is your IdP:
```
Comment (b-f-chan) [inline INSTALL.md:138]: Spelling: "For example **_if_** Google is your IdP:"

File: INSTALL.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
+# Installation Guide
+
+## Prerequisites
```
Comment (b-f-chan) [inline INSTALL.md:3]: Just for my understanding when putting this into the User docs.  Are these pre-requisites we can reasonably assume this audience (which is very technical) knows how to do (e.g. Postgres SQL installation, IdP client setup), or do I need to provide steps or references for them?

File: INSTALL.md [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+    - name: <name_of_your_ui_app>
+      type: CLIENT
+      clientId: <client_id_of_ui>
+      clientSecret: <some_secret> 
+      redirectUri: <url to redirect to in UI, for Ego UI this would be root>
+      description: Some description about this application  # optional
+```
+
+### First User
+Users by default do not have the `ADMIN` role and therefor cannot modify Ego or use the Ego UI. We want to allow the first user to login to be an ADMIN user. We can do that by using the following configuration in `application.yml`:
```
Comment (b-f-chan) [inline INSTALL.md:172]: Spelling: "and _**therefore**_"

File: INSTALL.md [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+
+### Putting it together.
+Now that we have updated our config we can restart Ego, and try to login via the UI.
+```shell
+bin/ego restart
+```
+
+## Cleanup
+
+Assuming all is well, and the Ego database is properly configured and the first user and application are working, you will most likely want to disable the initialization of the first application and user.
```
Comment (b-f-chan) [inline INSTALL.md:192]: Just so I'm clear, you mean afterwards going back into **_application.yml_** and setting **_firstUserAsAdmin_** back to false, correct?

**Review by b-f-chan — CHANGES_REQUESTED** [review]
Few minor spelling corrections + couple general questions

File: INSTALL.md (hunk 3 above)
Comment (andricDu) [reply INSTALL.md:3]: Whether or not they know how to do it is irrelevant to us. We are not supporting end users in their own infrastructure.

File: INSTALL.md (hunk 5 above)
Comment (andricDu) [reply INSTALL.md:192]: Yes

Comment (andricDu) [reply INSTALL.md]: Added text.

**Comment by andricDu** [comment]
@b-f-chan Thanks for the look over, changes made.

**Pull request merged**